vertical-pod-autoscaler/pkg/recommender/input/cluster_feeder.go

```diff
@@ -49,7 +49,14 @@ import (
 	resourceclient "k8s.io/metrics/pkg/client/clientset/versioned/typed/metrics/v1beta1"
 )
 
-const defaultResyncPeriod time.Duration = 10 * time.Minute
+const (
+	betweenScaleSubresourceRefreshes time.Duration = 10 * time.Minute
```

Comment:
Maybe you could consider running this more frequently so the load (GET calls to API server) is spread more evenly in time? If GetKeysToRefresh is lightweight (and it seems so), it might be beneficial to run the whole refresher subroutine more often than main loop of the recommender.

Reply:
Actually this should belong below.

Reply:
Done.